config/v1/0000_03_marketplace-operator_01_operatorhub.crd.yaml

```diff
@@ -3,7 +3,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.openshift.io: https://github.com/openshift/api/pull/470
-    capability.openshift.io/name: marketplace
+    capability.openshift.io/name: marketplace+OLM
```

Comment:
Does this need to be changed if we are ensuring that OLM is always enabled when a cluster has marketplace enabled?

What is the effect of this on a cluster that upgrades into 4.15?

Reply:
After testing it, that shouldn't be needed, I assumed given enhancement docs we might want to have it to make explicit in the capability name that a dependency exists, but this change didn't even seem to alter the payload CR anyway. I went ahead and removed it. Regarding the upgrades into 4.15, I added an upgrade validation comment. That behaved as expected as well, meaning the `ClusterVersion` was updated and the events showed the enforcement logic working.